### PR TITLE
fix upgrade issue to check common api to many versions

### PR DESCRIPTION
**Pull request by lihongyan1**
Fix issue by checking an API which is applicable to many releases.
https://issues.redhat.com/browse/OCPQE-522
Have tested on both 4.5 and 4.4

### CODE REVIEW

**Comment by lihongyan1** [comment]
@juzhao @pruan-rht Please help review

File: features/upgrade/monitoring/monitoring-upgrade.feature [hunk 1]
```diff
@@ -60,18 +60,18 @@ Feature: cluster monitoring related upgrade check
     And the output should contain:
       | Watchdog |
 
-    # curl -k -H "Authorization: Bearer $token" 'https://grafana.openshift-monitoring.svc:3000/api/search?folderIds=1
+    # curl -k -H "Authorization: Bearer $token" 'https://grafana.openshift-monitoring.svc:3000/api/health
```
Comment (juzhao) [inline features/upgrade/monitoring/monitoring-upgrade.feature:63]: remove the left `'`

**Comment by lihongyan1** [comment]
@juzhao @pruan-rht please help review

**Pull request merged**